frontend/src/scenes/billing/billingLogic.ts

```diff
@@ -0,0 +1,23 @@
+import { kea } from 'kea'
+import api from 'lib/api'
+
+export const billingLogic = kea({
```

Comment:
you should import `billingLogicType` here and pass it to `kea<billingLogicType>()`